core/variant/callable.cpp

```diff
@@ -36,11 +36,11 @@
 #include "core/object/ref_counted.h"
 #include "core/object/script_language.h"
 
-void Callable::call_deferred(const Variant **p_arguments, int p_argcount) const {
+void Callable::call_deferredv(const Variant **p_arguments, int p_argcount) const {
 	MessageQueue::get_singleton()->push_callablep(*this, p_arguments, p_argcount);
 }
 
-void Callable::call(const Variant **p_arguments, int p_argcount, Variant &r_return_value, CallError &r_call_error) const {
+void Callable::callv(const Variant **p_arguments, int p_argcount, Variant &r_return_value, CallError &r_call_error) const {
```

Comment:
Why `callv` not `callp`? (like in #58929)

Reply:
Ah you are right, I messed up.